### PR TITLE
Updated README 

### PR DESCRIPTION
**Pull request by jasonpraful**
Updated README to ensure proper instructions for Non-TypeScript implementation.
- Added instructions for login.js
- exported axiosinstance in api.js
- Added dependencies 'setAccessToken,etc' for JS implementation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -153,13 +153,15 @@ applyAuthTokenInterceptor(axiosInstance, {
 ## Non-TypeScript implementation
 
 ```javascript
-import { applyAuthTokenInterceptor } from 'react-native-axios-jwt';
-import axios from 'axios';
+//api.js
+
+import { applyAuthTokenInterceptor } from "react-native-axios-jwt";
```
Comment (mvanroon) [inline README.md:158]:
```suggestion
import { applyAuthTokenInterceptor } from 'react-native-axios-jwt';
```

File: README.md [hunk 2]
```diff
@@ -171,20 +173,43 @@ const requestRefresh = async (refresh) => {
...
+```javascript
+//login.js
+
+import {
+  isLoggedIn,
+  setAuthTokens,
+  clearAuthTokens,
+  getAccessToken,
+  getRefreshToken,
+} from "react-native-axios-jwt";
```
Comment (mvanroon) [inline README.md:188]:
```suggestion
} from 'react-native-axios-jwt';
```

File: README.md [hunk 3]
```diff
@@ -171,20 +173,43 @@ const requestRefresh = async (refresh) => {
...
+import {axiosInstance} from "../api";
```
Comment (mvanroon) [inline README.md:189]:
```suggestion
import { axiosInstance } from '../api';
```

**Review by mvanroon — CHANGES_REQUESTED** [review]

**Comment by mvanroon** [comment]
Thank you for your efforts

**Comment by jasonpraful** [comment]
Updated as requested

File: README.md [hunk 4]
```diff
@@ -171,20 +173,44 @@ const requestRefresh = async (refresh) => {
...
-  await setAuthTokens({
+    await setAuthTokens({
     accessToken: response.data.access_token,
     refreshToken: response.data.refresh_token
   })
 }
 
-// 5. Logging out
-const logout = async () => await clearAuthTokens()
+// 5. Clear the auth tokens from AsyncStorage
```
Comment (mvanroon) [inline README.md:202]: I just made a couple of tiny changes to the readme myself to clarify the code examples a bit more.

```suggestion
// 5. Log out by clearing the auth tokens from AsyncStorage
```

File: README.md [hunk 5]
```diff
@@ -171,20 +173,44 @@ const requestRefresh = async (refresh) => {
...
+import {
+  isLoggedIn,
+  setAuthTokens,
+  clearAuthTokens,
+  getAccessToken,
+  getRefreshToken,
+} from 'react-native-axios-jwt';
+import { axiosInstance } from '../api';
+
+// 4. Post email and password and get tokens in return. Call setAuthTokens with the result.
```
Comment (mvanroon) [inline README.md:191]:
```suggestion
// 4. Log in by POST-ing the email and password and get tokens in return
// and call setAuthTokens with the result.
```

**Review by mvanroon — CHANGES_REQUESTED** [review]

**Comment by mvanroon** [comment]
Cheers

**Pull request merged**